src/api/controllers/ContactFormController.js

```diff
@@ -6,24 +6,47 @@ module.exports = {
       firstName: req.body.firstName,
       lastName: req.body.lastName,
       email: req.body.email,
-      message: req.body.message
+      message: req.body.message,
     });
 
     contactForm
       .save()
       .then(() => {
-        res.status(201).send();
+        return res.status(201).send();
       })
-      .catch(err => {
+      .catch((err) => {
         if (err.errors.email) {
           if (err.errors.email.kind === "regexp") {
-            res.status(500).json({ error: "Invalid email address" });
+            return res.status(400).json({ error: "Invalid email address" });
+          } else if (err.errors.email.kind === "required") {
+            return res.status(400).json({ error: "Field `email` is required" });
           }
-          res.status(500).json(err);
+          return res.status(500).json(err);
```

Comment:
Don't return the error here.
```suggestion
          return res.status(500).json();
```